SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchPreview/AssetHistoryTable.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import moment from 'moment';
+import { SEBrowser } from '../../../global';
+import Table from '@gpa-gemstone/react-table';
+
+const AssetHistoryTable: React.FC<SEBrowser.IWidget<any>> = (props) => {
+    const [historyData, setHistoryData] = React.useState<Array<any>>([]);
+    const [count, setCount] = React.useState<number>(10);
+    const [assetName, setAssetName] = React.useState<string>('');
+
+    React.useEffect(() => {
+        let handle = getHistoryData();
+        handle.done((data) => {
+            console.log(data); // Add this line to log the 'data' variable
+            setHistoryData(data);
+            // Assuming the first record's AssetName is the asset name you want
+            if (data.length > 0) setAssetName(data[0].AssetName);
+        });
+
+        return () => {
+            if (handle.abort != undefined) handle.abort();
+        }
+    }, [props.eventID, count]);
+
+    function getHistoryData() {
+        return $.ajax({
+            type: "GET",
+            url: `${homePath}api/OpenXDA/GetEventSearchHistory/${props.eventID}/${count}`,
+            contentType: "application/json; charset=utf-8",
+            dataType: 'json',
+            cache: true,
+            async: true
+        });
+    }
+
+    return (
+        <div className="card">
+            <div className="card-header">Asset History Table for {assetName}:
+                <select className='pull-right' value={count} onChange={(evt) => setCount(parseInt(evt.target.value))}>
```

Comment:
Add a label to this dropdown that clarifies what it's for (i.e., `Num of Events`).

Reply:
Done